django_extensions/management/commands/print_settings.py

```diff
@@ -26,34 +31,47 @@ def add_arguments(self, parser):
         )
         parser.add_argument(
             '--format',
-            default='simple',
+            default=DEFAULT_FORMAT,
             dest='format',
             help='Specifies output format.'
         )
         parser.add_argument(
             '--indent',
-            default=4,
+            default=DEFAULT_INDENT,
             dest='indent',
             type=int,
             help='Specifies indent level for JSON and YAML'
         )
+        parser.add_argument(
+            '--show-secrets',
+            default=DEFAULT_SECRETS,
```

Comment:
use `action='store_true'` like the `--fail` option that way we don't need `DEFAULT_SECRETS` and the cli parser knows how to do the "right-thing"

Reply:
Right. Thanks for the tip!